docs/01-overview/main-areas/telemetry/telemetry-02-logs.md

```diff
@@ -0,0 +1,551 @@
+
+![Pipeline Concept](./assets/logging-pipelines.drawio.svg)
+
+1. A central `tail` input plugin reads the application logs.
+
+2. The application logs are enriched by a `kubernetes` filter. Then, for every LogPipeline definition, a `rewrite_tag` filter is generated, which uses a dedicated `tag` with name `<logpipeline>.*`, followed by the custom configuration defined in the LogPipeline resource. You can add your own filters to the default filters.
+
+3. Based on the default and custom filters, you get the desired output for each `LogPipeline`.
+
+This approach assures a reliable buffer management and isolation of pipelines, while keeping flexibility on customizations.
+
+### Telemetry Operator
+
+The LogPipeline resource is managed by the Telemetry Operator, a typical Kubernetes operator responsible for managing the custom parts of the Fluent Bit configuration.
+
+![Operator resources](./assets/logging-resources.drawio.svg)
+
+The Telemetry Operator watches all LogPipeline resources and related Secrets. Whenever the configuration changes, it validates the configuration (with a [validating webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)) and generates a new configuration for the Fluentbit DaemonSet, where several ConfigMaps for the different aspects of the configuration are generated. Furthermore, referenced Secrets are copied into one Secret that is mounted to the DaemonSet as well.
+
+## Setting up a LogPipeline
+
+### Step 1: Create a LogPipeline and output
+
+1. To ship application logs to a new output, create a resource file of kind LogPipeline:
+    ```yaml
+    kind: LogPipeline
+      apiVersion: telemetry.kyma-project.io/v1alpha1
+      metadata:
+        name: http-backend
+    spec:
+      output:
+        http:
+          dedot: false
+          port: "80"
+          uri: "/"
+          host:
+            value: https://myhost/logs
+          user:
+            value: "user"
+          password:
+            value: "not-required"
+    ```
+    An output is a data destination configured by a [Fluentbit output](https://docs.fluentbit.io/manual/pipeline/outputs) of the relevant type. The LogPipeline supports the following output types:
+
+    - **http**, which sends the data to the specified HTTP destination. The output is designed to integrate with a [Fluentd HTTP Input](https://docs.fluentd.org/input/http), which opens up a huge ecosystem of integration possibilities.
+    - **grafana-loki**, which sends the data to the Kyma-internal Loki instance. Note: This output is considered legacy and is only provided for backwards compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) in-cluster Loki instance. It might not be compatible with latest Loki versions. For integration with a custom Loki installation use the `custom` output with name `loki` instead, see also [this tutorial](https://github.com/kyma-project/examples/tree/main/loki).
```

Comment:
```suggestion
    - **grafana-loki**, which sends the data to the Kyma-internal Loki instance. 
    > **Note:** This output is considered legacy and is only provided for backward compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) in-cluster Loki instance. It might not be compatible with the latest Loki versions. For integration with a custom Loki installation, use the `custom` output with the name `loki` instead. See also [this tutorial](https://github.com/kyma-project/examples/tree/main/loki).
```